Content.Server/Atmos/Piping/Unary/EntitySystems/GasThermoMachineSystem.cs

```diff
@@ -90,7 +93,13 @@ private void OnThermoMachineUpdated(EntityUid uid, GasThermoMachineComponent the
                 thermoMachine.HysteresisState = false; // turn off
             }
             float dQActual = dQ * scale;
+            float dQLeak = dQActual * thermoMachine.EnergyLeakPercentage;
+            float dQPipe = dQActual - dQLeak;
```

Comment:
this isnt used, should dqpipe not go into inlet.air

Reply:
Good catch, the line below should be `dQPipe`.